pandas/core/arrays/base.py

```diff
@@ -1161,7 +1161,7 @@ def _create_method(cls, op, coerce_to_dtype=True):
         --------
         Given an ExtensionArray subclass called MyExtensionArray, use
 
-        >>> __add__ = cls._create_method(operator.add)
+        >>> __add__ = cls._create_method(operator.add)  # doctest: +SKIP
```

Comment:
```suggestion
            __add__ = cls._create_method(operator.add)
```

(this is not a runnable code example, rather how it should be coded in a class definition, so this also doesn't need a `>>>` prompt)